### PR TITLE
lesson1

### PR DESCRIPTION
**Pull request by Polik32**

### CODE REVIEW

File: Taras-Polonskiy/lesson1/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	<h3><p>Honesty, responsibility, attention to detail.<br>
+        Discipline, punctuality, sociability.</p></h3>
+	<h2>Additional Information:</h2>
+	<h3><ul>Marital status: Married.<br>
+        <li>Driving license: yes.</li><br>
+        <li>Actually a car: yes.</li><br>
+        <li>Possibility of business trips: yes.</li><br>
+		<li>Hobbies: Play sports, play football.</li><br></ul></h3>
+	</body>
+	</html>
```
Comment (AlexOrd) [inline Taras-Polonskiy/lesson1/index.html]: Hello

File: Taras-Polonskiy/lesson1/style.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+hr {
+	border: none;
+    background-color: #ff9582 ;
+    color: #ff9582;
+	height: 2px;
+	width:90%;
+}
+
+.foto{
```
Comment (AlexOrd) [inline Taras-Polonskiy/lesson1/style.css]: photo

File: Taras-Polonskiy/lesson1/index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	(20/06/2013 - 21/11/2013)</li><br><br>
+	<li>Vinnytsia City Council Social Policy Department, State Social Inspector<br>
+	(22/11/2013 - present)</li></ol></h3>
+	<h2>Personal qualities:</h2>
+	<h3><p>Honesty, responsibility, attention to detail.<br>
+        Discipline, punctuality, sociability.</p></h3>
+	<h2>Additional Information:</h2>
+	<h3><ul>Marital status: Married.<br>
+        <li>Driving license: yes.</li><br>
+        <li>Actually a car: yes.</li><br>
```
Comment (Valikbv) [inline Taras-Polonskiy/lesson1/index.html]: hey

File: Taras-Polonskiy/lesson1/index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+	Address: Vinnytsia<br>
+	Phone: 0677854875<br>
+	E-mail: polik19195@gmail.com<br>
+	<a class="facebook" href="https://www.facebook.com/profile.php?id=100004470759014">My Facebook profile</a></p></h3><br><br><br><br><br><br><br><br>
+	<h2>Education:</h2>
+	<h3><p>01/09/2004 - 06/30/2009, Vinnytsia State Pedagogical University named after M. Kotsubinsky Institute of History, Ethnology and Law, specialist diploma (full-time education)<br></p></h3>
+	<h2>Experience:</h2>
+	<h3><ol><li>Department of Labor and Social Protection of the WRDA, Chief Specialist<br>
+	(22/12/2009 - 19/06/2013)</li><br><br>
+	<li>Office of the Pension Fund of Ukraine in Vinnitsa, Chief Specialist<br>
```
Comment (AlexOrd) [inline Taras-Polonskiy/lesson1/index.html]: Todo

File: Taras-Polonskiy/lesson1/index.html (hunk 1 above)
Comment (avatar1807) [reply Taras-Polonskiy/lesson1/index.html]: HI

File: Taras-Polonskiy/lesson1/style.css [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+h3{
+	margin: 0 0 0 10px;
+}
+
+hr {
+	border: none;
+    background-color: #ff9582 ;
+    color: #ff9582;
+	height: 2px;
+	width:90%;
```
Comment (i1Gaming) [inline Taras-Polonskiy/lesson1/style.css]: add space

File: Taras-Polonskiy/lesson1/index.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+  <meta charset="utf-8" />
+  <link href="style.css" rel="stylesheet">
+  <title>Resume</title>
```
Comment (kasmatyn) [inline Taras-Polonskiy/lesson1/index.html]: try CV

**Pull request merged**